onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -25,6 +52,12 @@
 
 extern std::unique_ptr<Ort::Env> ort_env;
 
+#define ASSERT_ORT_STATUS_OK(function)   \
+  do {                                   \
+    OrtStatus* _tmp_status = (function); \
+    ASSERT_EQ(_tmp_status, nullptr);     \
```

Comment:
is it possible to include the status code and message in the assertion failure output?

Reply:
> should a non-nullptr _tmp_status be freed

It doesn't matter. This is a unit test. When an assert fails, the developer should stop there and debug it. We don't need to care if memory leak would happen after an assert failed.

Reply:
Error message added.